index.css

```diff
@@ -0,0 +1,130 @@
+* { box-sizing: border-box;}
+Body{
+  background-attachment: fixed;
+  background-image: url('http://www.fleming.kyschools.us/userfiles/316/Clipart%20Images/math.jpg');
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-position: center;
+  margin:auto;
+}
+.calculator-title{
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+.calculator-calcBody{
+  box-shadow: 30px 30px 30px grey;
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  align-items: center;
+  justify-content: center;
+  position: fixed;
+  border: 1px black solid;
+  background-color: grey;
+  border-width: medium;
+  border-radius: 10px;
+  top: 20%;
+  left:40%;
+  width: 20%;
+  height: 75%;
+}
+
+.calculator-valueDisplay{
+  /*display: flex;*/
+  /*justify-content: center;*/
+  /*align-items: center;*/
+  font-family: 'Roboto', sans-serif;
+  background-color: grey;
+  color: white;
+  border-radius: 7px 7px 0px 0px;
+  width: 100%;
+  height: 15%;
+  padding-top: 10px;
+  padding-right: 3px;
+  text-align: right;
+  font-size: 380%;
+  /*padding: 3px;*/
+}
+.calculator-row{
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  font-family: 'Roboto', sans-serif;
+  outline: none;
+}
+.calculator-button-floating-box {
+  font-size: 150%;
+  font-weight: bold;
+  background-color: #f2f2f2;
+  width: 24.999%;
```

Comment:
the calculator should not be quishy. it should not change size as the window changes size. It should be a fixed size in pixels. 
consider changing: 
```css
width: 24.999%;
height: 16.99%;
```

to fixed pixel values.